app/src/main/java/org/oppia/android/app/player/state/ImageRegionSelectionInteractionView.kt

```diff
@@ -66,6 +69,16 @@ class ImageRegionSelectionInteractionView @JvmOverloads constructor(
     maybeInitializeClickableAreas()
   }
 
+  fun setClickedRegions(clickedRegions: ClickOnImage) {
```

Comment:
Doesn't the initial state need to be set via an init{} block like it is for the other interactions?

Reply:
Yes we need to initialize state and I am still figuring out how can we initialize state for ImageRegionSelection